code/datums/components/larva_queue.dm

```diff
@@ -91,9 +91,13 @@
 /datum/component/larva_queue/proc/set_queue_position(waiter, new_position)
 	SIGNAL_HANDLER
 	position = new_position
+	var/datum/hive_status/the_hive = GLOB.hive_datums[XENO_HIVE_NORMAL]
```

Comment:
isn't this going to runtime if no hive is setup?

Reply:
Aren't there going to be normal hives always anyways? and if this is true, this means that the button will runtime every click at line (previously 113) 117.

Reply:
I unno, its mainly for before the game starts, but I wouldn't just trust existing code to believe whether it runtimes lol